mungegithub/mungers/approval-handler.go

```diff
@@ -107,11 +107,18 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 			int64(*obj.Issue.Number)),
 		approverSet}
 
+	assignees := []string{}
+	for _, user := range obj.Issue.Assignees {
+		if user != nil && user.Login != nil {
+			assignees = append(assignees, *user.Login)
+		}
+	}
+
 	notificationMatcher := c.MungerNotificationName(approvers.ApprovalNotificationName)
 
 	latestNotification := c.FilterComments(comments, notificationMatcher).GetLast()
 	latestApprove := getApproveComments(comments).GetLast()
-	newMessage := h.updateNotification(obj.Org(), obj.Project(), latestNotification, latestApprove, approversHandler)
+	newMessage := h.updateNotification(obj.Org(), obj.Project(), latestNotification, latestApprove, approversHandler, assignees)
```

Comment:
this is getting messy again.  Should we make assignees a field of approvers.Approvers instead?